subworkflows/nf-core/utils_nextflow_pipeline/main.nf

```diff
@@ -106,19 +110,13 @@ def checkCondaChannels() {
 
     required_channels_in_order.eachWithIndex { channel, index ->
         if (index < required_channels_in_order.size() - 1) {
-            channel_priority_violation |= !(channels.indexOf(channel) < channels.indexOf(required_channels_in_order[index+1]))
+            channel_priority_violation |= !(channels.indexOf(channel) < channels.indexOf(required_channels_in_order[index + 1]))
         }
     }
 
     if (channels_missing | channel_priority_violation) {
-        log.warn "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n" +
-            "  There is a problem with your Conda configuration!\n\n" +
-            "  You will need to set-up the conda-forge and bioconda channels correctly.\n" +
-            "  Please refer to https://bioconda.github.io/\n" +
-            "  The observed channel order is \n" +
-            "  ${channels}\n" +
-            "  but the following channel order is required:\n" +
-            "  ${required_channels_in_order}\n" +
-            "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+        log.warn(
+            "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n" + "  There is a problem with your Conda configuration!\n\n" + "  You will need to set-up the conda-forge and bioconda channels correctly.\n" + "  Please refer to https://bioconda.github.io/\n" + "  The observed channel order is \n" + "  ${channels}\n" + "  but the following channel order is required:\n" + "  ${required_channels_in_order}\n" + "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+        )
```

Comment:
I tried to preserve the formatting here but found it to be tricky, probably a multi-line string would be better anyway:
```groovy
    log.warn """\
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
          There is a problem with your Conda configuration!

          You will need to set-up the conda-forge and bioconda channels correctly.
          Please refer to https://bioconda.github.io/
          The observed channel order is 
          ${channels}
          but the following channel order is required:
          ${required_channels_in_order}
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        """.stripIndent(true)

```